packages/manager/apps/nasha-react/src/components/layout-helpers/dashboard/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { Outlet, NavLink, useLocation } from 'react-router-dom';
+import {
+  OsdsTabs,
+  OsdsTabBar,
+  OsdsDivider,
+  OsdsTabBarItem,
+} from '@ovhcloud/ods-stencil/components/react';
+import './dashboard.scss';
+
+export type DashboardTabItemProps = {
+  name: string;
+  title: string;
+  to: string;
+};
+
+export type DashboardLayoutProps = {
+  tabs: DashboardTabItemProps[];
+};
+
+const DashboardLayoutHelpers: React.FC<DashboardLayoutProps> = ({ tabs }) => {
+  const [panel, setActivePanel] = useState('');
+  const location = useLocation();
+
+  useEffect(() => {
+    const activeTab = tabs.find((tab) => tab.to === location.pathname);
+    if (activeTab) {
```

Comment:
if there is no activeTab we should set the active panel to `undefined` also, don't we ?